pom.xml

```diff
@@ -37,34 +37,45 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <serenity.version>4.0.8</serenity.version>
```

Comment:
Nit: It might be beneficial to move report portal related libraries versions here as well.